include/lsp-plug.in/dsp-units/sampling/Sample.h

```diff
@@ -176,6 +176,15 @@ namespace lsp
                  */
                 bool resize(size_t channels, size_t max_length, size_t length = 0);
 
+                /** Stretch sample
+                 *
+                 * @param channels number of channels
```

Comment:
The `Sample` is an audio buffer that defines the desired number of channels. Do we really need the `channels` parameter? I think this is the unnecessary function to also allow to change the number of channels for the audio sample.